v22.2/backup-and-restore-monitoring.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Backup and Restore Monitoring
+summary: An overview of the metrics to monitor backup and restore jobs in CockroachdB.
+toc: true
+docs_area: manage
+---
+
+CockroachDB includes metrics to monitor [backup](backup.html), [restore](restore.html), and [scheduled backup](create-schedule-for-backup.html) jobs. You can use monitoring integrations to alert when there are anomalies, such as backups that have failed or restore jobs encountering a retryable error.
+
+Depending on whether you are using a {{ site.data.products.dedicated }} or {{ site.data.products.core }} cluster, you can use the following to monitor backup and restore metrics for your CockroachDB cluster:
+
+- [Prometheus endpoint](#prometheus-endpoint): {{ site.data.products.dedicated }}, {{ site.data.products.core }}
+- [Datadog integration](#datadog-integration): {{ site.data.products.dedicated }}
+
+We recommend setting up monitoring to alert when anomalies occur. You can then use the following SQL statements to inspect details relating to schedules, jobs, and backups:
+
+- [`SHOW SCHEDULES`](show-schedules.html)
+- [`SHOW JOBS`](show-jobs.html)
+- [`SHOW BACKUP`](show-backup.html)
+
+For detail on [managed-service backups](../cockroachcloud/use-managed-service-backups.html) that Cockroach Labs stores for your {{ site.data.products.db }} cluster, see the **Backups** page in the Cloud Console.
+
+{% include {{ page.version.version }}/backups/metrics-per-node.md %}
+
+## Prometheus endpoint
+
+You can access the [Prometheus endpoint](monitoring-and-alerting.html#prometheus-endpoint) (`http://<host>:<http-port>/_status/vars`) for backup and restore metrics with **{{ site.data.products.dedicated }} or {{ site.data.products.core }}** clusters.
+
+See the [Monitor CockroachDB with Prometheus](monitor-cockroachdb-with-prometheus.html) tutorial for guidance on installing and setting up Prometheus and Alertmanager to track metrics.
+
+### Available metrics
+
+Metric | Description 
+-------+-------------
+`schedules_backup_succeeded` | The number of scheduled backup jobs that have succeeded.
+`schedules_backup_started` | The number of scheduled backup jobs that have started.
+`schedules_backup_last_completed_time` | The Unix timestamp of the most recently completed backup by a schedule specified as maintaining this metric.
```

Comment:
should we mention the parameter on the schedule that is used to specify this?

Reply:
Ah, @dt this parameter: `updates_cluster_last_backup_time_metric`... we don't currently document this. Was it because if CC customers use it on their scheduled backups it could mess with the CRL-run scheduled backups in some way? If that's the case, I still think we can document it here, might just have to come with that warning. That is, if the CC issue is a thing.

Reply:
I'll defer to @livlobo but I think we should doc it; I don't love the idea that we just hope nobody finds it as a strategy for CC , and it is generally useful to non-CC users for the same reason CC wanted it: this is _the_ thing you'd want to be monitoring -- short of having a tool go look in the buckets themselves for successful backups but that is annoying to do -- to know that you're backing up as expected.

Reply:
Gotcha I made some adjustments to see what you both think. For now, I have added in the option (see the screenshot below), can remove if @livlobo doesn't want to document. I may have gone a bit overboard on the warnings here, but it's a starting point at least, if we're going to document:
<img width="876" alt="image" src="https://user-images.githubusercontent.com/44557882/220446459-e51a7d37-fc20-4ed1-aec7-dd5c20394342.png">

Reply:
We really need to find a way to separate the managed service backup monitoring from the customer owned backup monitoring. For the problem of customers accidentally overwriting the manages service schedules, we said, we'd rely on monitoring. Now, I feel like we're exposing the monitoring to potential clobbering?

Reply:
@livlobo as part of another PR: https://github.com/cockroachdb/docs/pull/16316 on re-organizing the nav, the proposal is sections on managed service & customer-owned backups, how about this monitoring page (as it stands) lives under the customer-owned backup section? Then, this screenshot above, should actually say "For **customer-owned backups only**" ?

Reply:
I think we're worrying about the potential for clobbering way too much without evidence that it will happen and/or is a problem if it does happen. I don't even know if we need the warnings here. I'd just say let's document the facts of the flag: if set, the schedule will update this metric, which is a good way to monitor that backups are succeeding as expected. I don't think we need to talk one way for cloud and another elsewhere or warn people about anything.